packages/client/src/StreamrClient.ts

```diff
@@ -564,6 +564,11 @@ export class StreamrClient {
         return this.node.getNode()
     }
 
+    async inspect(node: NetworkPeerDescriptor, streamDefinition: StreamDefinition): Promise<boolean> {
```

Comment:
This will return `true` if `node` is in the stream and propagating messages and `false` otherwise, right?

Is there a circumstance where this will reject, perhaps on timeout?

Reply:
You can never be sure if a rejection happens for some reason, but the function will return false on inspection timeouts.